### ISSUE TITLE
Segmentation fault in Catalina 10.15.2

### ISSUE THREAD

**Issue by henrik242** [jakehilborn/displayplacer#21]
I just copypasted the modeline from the `list` output, and got a segmentation fault:
```$ displayplacer "id: AC430A46-436F-287B-3CD7-0BC5BFDE526F mode 245: res:2048x1280 hz:59 color_depth:8 scaling:on"
Segmentation fault: 11
```
[list.txt](https://github.com/jakehilborn/displayplacer/files/4126942/list.txt)

So this is probably because of my bad formatting of the modeline :)  Maybe give a proper error instead of a segfault, and format the `list` output for better copypasting?

`displayplacer "id:AC430A46-436F-287B-3CD7-0BC5BFDE526F res:2048x1280 hz:59 color_depth:8 scaling:on origin:(0,0) degree:0"` works fine.

**Comment by jakehilborn** [comment]
Yeah it's because of bad formatting. All the input is parsed manually instead of using a library like [argparse](https://github.com/Cofyc/argparse) with more resiliency. It's a small annoyance so it's not something I intend to refactor. I'm happy to accept and review pull requests though.
